modules/account-lib/src/coin/algo/utils.ts

```diff
@@ -547,6 +547,20 @@ export class Utils implements BaseUtils {
     }
     return type;
   }
+
+  /**
+   * Validate if the key is a valid base64 string
+   * @param key the key to validate
+   */
+  validateBase64(key: string): void {
+    if (!key || typeof key !== 'string') {
+      throw new Error('Invalid base64 string');
+    }
+    const base64RegExp = new RegExp('^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==\|[A-Za-z0-9+/]{3}=)?$')
```

Comment:
you can make this a constant outside function to not compile it each time is used. Also the regex can be created as a regex literal:
```suggestion
    const base64RegExp = /^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==\|[A-Za-z0-9+/]{3}=)?$/
```